.github/workflows/cibuild.yml

```diff
@@ -21,6 +21,11 @@ jobs:
       fail-fast: false
       matrix:
         configuration: [msvc-debug, msvc-release]#, clang-debug]
+        enable_steam: [false]
```

Comment:
perhaps ON/OFF instead of true/false? ON/OFF is shorter but GitHub actions do not understand them as boolean values.